examples/lighting-app/efr32/src/gen/callback-stub.c

```diff
@@ -1997,6 +1997,19 @@ bool emberAfReadReportingConfigurationCommandCallback(const EmberAfClusterComman
     return false;
 }
 
+/** @brief Activate Door Lock Callback
```

Comment:
I wouldn't expect this function to be needed here.... Is it needed? Same for the callback.h in this dir.